contracts/license.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.5.0;
+
+import "./ownable.sol";
+
+contract License is Ownable {
+    address payable recipient;
+    uint256 price;
+
+    mapping(address => uint256) licenseOwners;
+
+    constructor(address payable _recipient, uint256 _price) public {
+        recipient = _recipient;
+        price = _price;
+    }
+
+    function isLicenseOwners() public view returns (bool) {
```

Comment:
It would be a good idea to receive the address to verify if it's a licensed owner, in case this contract is called from another contract